deployment/dynatrace-azure-logs.sh

```diff
@@ -86,14 +86,31 @@ check_arg()
     fi
 }
 
+check_api_token() {
+  if RESPONSE=$(curl -k -s -X POST -d "{\"token\":\"$TARGET_API_TOKEN\"}" "$TARGET_URL/api/v2/apiTokens/lookup" -w "<<HTTP_CODE>>%{http_code}" -H "accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $TARGET_API_TOKEN"); then
+    CODE=$(sed -rn 's/.*<<HTTP_CODE>>(.*)$/\1/p' <<<"$RESPONSE")
+    RESPONSE=$(sed -r 's/(.*)<<HTTP_CODE>>.*$/\1/' <<<"$RESPONSE")
+    if [ "$CODE" -ge 300 ]; then
+      echo "Failed to check Dynatrace API token permissions - please verify provided values for parameters: --target-url and --target-api-token. $RESPONSE"
+      exit 1
+    fi
+    if ! grep -q "logs.ingest" <<<"$RESPONSE"; then
```

Comment:
I'm wondering if we should wrap "logs.ingest" with another quotes to match exact JSON token `"logs.ingest"` and avoid false positives?